authui/src/authflowv2/components/primary-button.css

```diff
@@ -1,45 +1,78 @@
 @layer components {
+  :root {
+    --primary-btn-bg-color: var(--c-primary-base);
+    --primary-btn-text-color: var(--c-white);
+    --primary-btn-border-radius: var(--border-radius-rounded-square);
+    --primary-btn-font-family: var(--font-family-label-large);
+    --primary-btn-font-size: var(--font-size-label-large);
+    --primary-btn-line-height: var(--line-height-label-large);
+    --primary-btn-letter-spacing: var(--letter-spacing-label-large);
+    --primary-btn-font-weight: var(--font-weight-label-large);
+    --primary-btn-ring-width: 0px;
+    --primary-btn-ring-color: transparent;
+    --primary-btn-py: var(--primary-btn-font-size);
+
+    --primary-btn-px: calc(
+      (var(--primary-btn-py) * 2 + var(--primary-btn-line-height)) / 2
+    ); /* = Height / 2 */
+
+    --primary-btn-bg-color-hover: var(--c-primary-500);
```

Comment:
I suggest we borrow the idea of BEM https://getbem.com/naming/ in naming our variables.

Examples

`--primary-btn__bg-color--hover`, `--primary-btn__px`

Block in BEM is our component name.
Element in BEM is the CSS property name of the component.
Modifier in BEM is the same.